src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+@Entity
 public class Actor implements Cloneable {
```

Comment:
Implementing Cloneable and overriding `clone` method is not typical for JPA entities. Entities should not be cloned in this manner due to the potential for breaking entity identity guarantees and causing issues with the persistence context. If cloning functionality is needed, consider using a copy constructor or a dedicated method for creating a copy without using `Cloneable`.